grpc/service_config/service_config.proto

```diff
@@ -220,10 +231,15 @@ message XdsConfig {
 
 // Selects LB policy and provides corresponding configuration.
 //
-// In general, all instances of this field should be repeated.
-// Clients will iterate through the list in order and stop at the first
-// policy that they support.  This allows the service config to specify
-// custom policies that may not be known to all clients.
+// In general, all instances of this field should be repeated. Clients will
+// iterate through the list in order and stop at the first policy that they
+// support.  This allows the service config to specify custom policies that may
+// not be known to all clients.
+//
+// - If the config for the first registered policy is invalid, the whole service
```

Comment:
We use the term "supported" in the previous paragraph, but we use the term "registered" in these two bullets.  I think either term is fine, but let's be consistent.

Reply:
Changed to `supported`.